apps/extension/src/webview/index.tsx

```
@@ -0,0 +1,1709 @@
+				<div className="p-4 border-b border-vscode-border">
+					<h2 className="text-lg font-semibold text-vscode-foreground">
+						Edit Task #{task.id}
+					</h2>
+				</div>
+
+				<form onSubmit={handleSubmit} className="p-4 space-y-4">
+					<div>
+						<label className="block text-sm font-medium text-vscode-foreground mb-2">
+							Title
+						</label>
+						<input
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)